tests/unit/clients/redis_cluster_tests.py

```diff
@@ -1,9 +1,119 @@
+import os
 import unittest
 
+from unittest import mock
+
 import fakeredis
+import pytest
+
+from prometheus_client import REGISTRY
+
+try:
+    import redis
```

Comment:
Shouldn't this be trying to import rediscluster?

Not happy about this style of skipping tests just because something the lib provides is not installed but you didn't introduce the pattern so I'm not going to block the PR for now. It runs the risk of running a test pass on baseplate and tests pass but not these, which you wouldn't know because you forgot to install something.

Reply:
i didn't introduce the pattern but i'll try to remove this